File: Day_2/Q6.java
import java.util.*;
class Q6{
    //Calculating Reverse
    static int reverse(int n){
        int rev = 0;
        while(n!=0){
            rev = (rev*10) + (n%10);
            n/=10;
        }
        return rev;
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        //Printing Result
        System.out.println("The Reverse Of "+n+" Is "+reverse(n));
        sc.close();
    }
}

File: Day_2/Q6.cpp
#include <iostream>
using namespace std;
int reverse(int n){
    //Calculating Reverse
    int rev = 0;
    while(n!=0){
        rev = (rev*10) + (n%10);
        n/=10;
    }
    return rev;
}
int main(){
    int n;
    cin >> n;
    //Printing Result
    cout << "The Reverse Of " << n << " Is " << reverse(n);
    return 0;
}